src/main/java/io/vertx/ext/stomp/impl/StompServerTCPConnectionImpl.java

```diff
@@ -91,6 +93,11 @@ public String session() {
     return sessionId;
   }
 
+  @Override
+  public SSLSession sslSession() {
+    return this.socket.sslSession();
```

Comment:
Either this or a getter like:

```
public NetSocket socket() {
   // ...
}
```

I tend to prefer tour approach as it avoids giving access to the socket to the users when they can manipulate it (in a non-stomp compliant manner).

Reply:
lfmunoz4@gmail.com can be used to verify my eclipse CLA